ZenPacks/zenoss/Microsoft/Windows/modeler/WinRMPlugin.py

```diff
@@ -144,6 +148,8 @@ def log_error(self, log, device, error):
                             'http://wiki.zenoss.org/ZenPack:Microsoft_Windows#winrm_setup'
         elif isinstance(error, ConnectionRefusedError):
             message = "Connection refused on %s: Verify WinRM setup"
+            evt_message = "Connection refused on %s: Verify WinRM setup" % device.id
```

Comment:
It's redundant to include the device id in the event's message or summary since there's already an event field for it.